ctf/pipes_and_punters/map.xml

```diff
@@ -49,8 +49,8 @@
     <kit id="map-spawn-kit" parent="default-kit">
         <kit id="spawn-kit"/>
         <kit id="resistance-kit">
-            <knockback-reduction>0</knockback-reduction>
-            <effect amplifier="5">damage resistance</effect>
+            <knockback-reduction>1</knockback-reduction>
+            <effect duration="4s" amplifier="255">damage resistance</effect>
         </kit>
     </kit>
     <take kit="resistance-kit">

```

Comment:
Please address this if you're removing infinite resistance.

Reply:
Alternatively, leave this as is unless there's a reason that resistance should not be infinite until leaving spawn given the map layout and just fix the knockback resistance.

Reply:
The reason is evident, you can camp spawn with the flag and be literally immortal while holding the flag

Reply:
> Please address this if you're removing infinite resistance.

Removed `resistance-kit` and merged knockback reduction + resistance into `spawn-kit`. On spawn exit, players now lose their knockback reduction but will not lose resistance as it is now for a limited duration.